### PR TITLE
Ben_Baker_Control_Flow

### PR DESCRIPTION
**Pull request by bbakercello**

### CODE REVIEW

**Comment by bbakercello** [comment]
Hi Brian, I made more changes after making a pull request. How do I update again with another pull request?

File: lib/challenge.js [hunk 1]
```diff
@@ -227,79 +243,98 @@ let nums = [
...
+for(let i=0; i <= 0; i++){
+console.log(nestedArrays[0])
 
-// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
+for(let i=0; i <= 0; i++){
+  console.log(nestedArrays[1])
+  }
+  for(let i=0; i <= 0; i++){
+    console.log(nestedArrays[2])
+    }
```
Comment (krabecb) [inline lib/challenge.js:313]: Inner loop should have a different iteration variable, such as `j`

Comment (krabecb) [reply lib/challenge.js:313]: That way, we can have syntax like `[i][j]`.

**Comment by krabecb** [comment]
Good effort on this deliverable! Excellent work with git commits! Your PR is still open so I am able to see your updates! Closing PR and marking "complete".

-BK 07/29

**Pull request closed without merging**